test/src/test/java/org/corfudb/integration/LogReplicationIT.java

```diff
@@ -1418,6 +1477,7 @@ public TestConfig clear() {
             writingDst = false;
```

Comment:
As part of the cleanup, it does not look like `trim` or `writingDst` are used anywhere